File: Dart/random_user.dart
import 'dart:convert';
import 'package:http/http.dart' as http;

main() async {
  // cerner_2^5_2018
  var inc = 'gender,name,nat';
  var requestUrl = 'https://randomuser.me/api/?inc=$inc';
  var response = await http.get(requestUrl);

  var jsonResponse = JSON.decode(response.body);

  var firstResult = jsonResponse['results'][0];
  var name = firstResult['name'];
  var nat = firstResult['nat'];
  print('$name : $nat');
}

File: F#/RandomUser.fs
// cerner_2^5_2018

string html = string.Empty;
string url = @"https://randomuser.me/api/?inc=gender,name,nat";

HttpWebRequest request = (HttpWebRequest)WebRequest.Create(url);
request.AutomaticDecompression = DecompressionMethods.GZip;

using (HttpWebResponse response = (HttpWebResponse)request.GetResponse())
using (Stream stream = response.GetResponseStream())
using (StreamReader reader = new StreamReader(stream))
{
    html = reader.ReadToEnd();
}

Console.WriteLine(html);